docs/development/extensions-core/stats.md

```diff
@@ -107,6 +107,18 @@ To acquire standard deviation from variance, user can use "stddev" post aggregat
 
 ### Timeseries query
 
+#### Druid SQL
+
+```SQL
```

Comment:
It will also be nice to include a mention of the SQL functions `VARIANCE` and `STDDEV` in the above section where it provides the native JSON syntaxes